client/Hooks/README.md

```diff
@@ -26,4 +26,4 @@ Where:
 
 ## Implemented hooks
 
-Hooks to be added here on implementation, with summary of purpose/usage and a link to it's README.
+- [useRouteConfig](./useRouteconfig/README.md) - Used to convert a page configuration into a useable React-router configuration.
```

Comment:
I did have that thought, I just went with a blanket 'Hooks belong in the Hooks' folder approach. Don't disagree that it could easily be part of Nav structure. Although then we would be going against our own approach that 'Bootstrap should only be tested through e2e' :P

Reply:
Happy to see what the concensus is - I would still vote it should be in with the Navigation myself (unless there is a use case where it is needed eslewhere), and have the tests alongside - no harm in them being there (perhaps no tests are expected, but can be provided if suitable)?